src/codegen/codegen_cpp_visitor.cpp

```diff
@@ -524,6 +529,18 @@ void CodegenCppVisitor::print_function_call(const FunctionCall& node) {
     printer->add_text(')');
 }
 
+void CodegenCppVisitor::print_nrn_pointing(const ast::FunctionCall& node) {
+    const auto& args = node.get_arguments();
+    if (args.size() != 1) {
+        throw std::runtime_error(
+            fmt::format("nrn_pointing excepts exactly one argument, got: {}", args.size()));
+    }
```

Comment:
just a nitpick: checks like this are typically done in [the semantic analysis visitor](https://github.com/BlueBrain/nmodl/blob/2f448528977bd8b60e552123fc7195bd35cecf2b/src/visitors/semantic_analysis_visitor.hpp#L39).

Reply:
ty, changed.